### PR TITLE
Exam

### PR DESCRIPTION
**Pull request by konik56**

### CODE REVIEW

File: Tasks/distinct.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-        } else {
-        A.add(a)
-        };
-        w++;
-    });
-  return data;}
+const leaveUniqueValues = function(array) {
+  if (!Array.isArray(array)) {
+    return;
+  };
```
Comment (tshemsedinov) [inline Tasks/distinct.js:7]:
```suggestion
  if (!Array.isArray(array)) return;
```

File: Tasks/merge.js [hunk 2]
```diff
@@ -1,19 +1,32 @@
...
-        object_3[attribute_name] = object_1[attribute_name]
-  }
-   for (attribute_name in object_2) {
-      (object_3[attribute_name] == object_2[attribute_name])
-     object_3[attribute_name] = object_2[attribute_name]
-    }
-return object_1, object_2, object_3;
-})
+const mergeTwoObjects = function(objFirst, objSecond) {
+  let objFinal = {};
```
Comment (tshemsedinov) [inline Tasks/merge.js:5]: Why not const?

File: Tasks/merge.js [hunk 3]
```diff
@@ -1,19 +1,32 @@
...
 
-const result = merge_two_objects({ a: 'uno', b: 'due' }, { c: 'tre' });
+  for (const key in objFirst) {
```
Comment (tshemsedinov) [inline Tasks/merge.js:7]: Prefer for..of

File: Tasks/merge.js [hunk 4]
```diff
@@ -1,19 +1,32 @@
...
+    };
+  };
+
+  return objFinal;
+};
+
+const result = mergeTwoObjects({ a: 'uno', b: 'due' }, { c: 'tre' });
 console.log(result);
+
+//Way what is not supported with some browsers
```
Comment (tshemsedinov) [inline Tasks/merge.js:25]: Supported everywhere

File: Tasks/merge.js (hunk 2 above)
Comment (konik56) [reply Tasks/merge.js:5]: my mistake, forgot about meaning of const is we have link

File: Tasks/merge.js (hunk 3 above)
Comment (konik56) [reply Tasks/merge.js:7]: (for (const key of Object.keys(objFirst)) {)- like this ?

**Pull request closed without merging**